controller/disk.go

```diff
@@ -36,6 +36,9 @@ func DisksCheck() (disks []model.Disk) {
 		dfCommand := ExecCommand("df -B1 /dev/" + diskName)
 		dfResult := dfCommand.String()
 		disk, err := parseDf(dfResult)
+		disk.Total = disk.Total + " Kb"
```

Comment:
Here would not be B (bytes)? Or maybe make sense to convert the value to Kb first, and after use a unit pattern like Kb.

Reply:
I fixed this changing the df command to show spaces in kb [here](https://github.com/robertoduessmann/health-check/pull/11/commits/e4e7abb3b5f93df2ff4653449c5150dd4c7c1119).